sql-statements/sql-statement-admin-cleanup.md

```diff
@@ -10,7 +10,7 @@ The `ADMIN CLEANUP INDEX` statement is used to delete redundant indexes from a t
 ## Synopsis
 
 ```ebnf+diagram
-AdminCleanupStmt ::=
+AdminStmt ::=
```

Comment:
I think it should only be called `AdminStmt` if it describes the full `ADMIN...` statement